ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/VertxNonceRequestTransmitter.java

```diff
@@ -117,8 +118,7 @@ private void handleResponse(final Buffer bodyBuffer, final CompletableFuture<Big
     } catch (final DecodeException e) {
```

Comment:
since we are not doing any thing with decode exception, may be log a debug level exception message?

Reply:
removed